frontend/src/app/testResults/resultsTable/ResultsTable.tsx

```diff
@@ -58,55 +58,57 @@ export const generateTableHeaders = (
 );
 
 function createActionItemList(
-  setPrintModalId: SetStateAction<String>,
+  setPrintModalId: Function,
```

Comment:
using `Function` type is too generic, we would lose lots of the benefits of type checking.
I recommend using `Dispatch<SetStateAction<undefined>>`
you can also define a custom type for that and use it throughout the app, if it looks pretty ugly 
`export type stateHook = Dispatch<SetStateAction<string | null>>`

another refactor might be also consolidating all these initial states of undefined, string | null to the same thing
https://github.com/CDCgov/prime-simplereport/blob/7604776c281900071ea1733d355567438fd5040b/frontend/src/app/testResults/TestResultsList.tsx#L161-L171

Reply:
I would rather not do extensive refactors because this effort was actually not aligned and started as me wanting to remove the "any" usage in ResultsTable.tsx component. 

I avoided using the Dispatch<SetStateAction<string | undefined>> because it requires more code changes but I can do that change as the scope is still relatively small 👍.

Reply:
Done!